static/manifest.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "brid.gy",
+  "short_name": "brid.gy",
+  "description": "📣 Connects your web site to social media. Likes, retweets, mentions, cross-posting, and more...",
+  "id": "/",
+  "start_url": "/",
+  "scope": "/",
+  "lang": "en",
+  "dir": "ltr",
+  "theme_color": "#428ef4",
+  "background_color": "#ffffff",
+  "display": "standalone",
+  "orientation": "portrait-primary",
+  "related_applications": [],
```

Comment:
probably remove these two?

Reply:
Not recommend or necessary

Reply:
Not necessary, but I'd like to, since they're optional 😁 https://developer.mozilla.org/en-US/docs/Web/Manifest#members

Reply:
Misunderstanding here from my side :D. I meant: They are not necessary, because optional